### PR TITLE
infinispan server tutorial

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: blog/2020-07-28-infinispan-server-tutorial.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+date: '2020-07-28T12:00:00.000-00:00'
+author: Katia Aresti
+tags: [ "tutorial", "learning", "server" ]
+---
+
+= Infinispan Server Tutorial
+
+Dear Infinispan community,
+
+If you are wondering how to start with the Infinispan Server from your Java Application,
```
Comment (oraNod) [inline blog/2020-07-28-infinispan-server-tutorial.adoc:13]:
```suggestion
If you are wondering how to start with Infinispan Server from your Java Application,
```

File: blog/2020-07-28-infinispan-server-tutorial.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+a new https://infinispan.org/infinispan-server-tutorial/[Tutorial] is available.
+This tutorial covers the essentials to get started with the Infinispan Server and
```
Comment (oraNod) [inline blog/2020-07-28-infinispan-server-tutorial.adoc:15]:
```suggestion
This tutorial covers the essentials to get started with Infinispan Server and
```

File: blog/2020-07-28-infinispan-server-tutorial.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+takes around a half an hour to be completed.
```
Comment (oraNod) [inline blog/2020-07-28-infinispan-server-tutorial.adoc:16]:
```suggestion
takes around half an hour to complete.
```

File: blog/2020-07-28-infinispan-server-tutorial.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+In this tutorial you will learn how to:
+
+. Run the Infinispan Server.
```
Comment (oraNod) [inline blog/2020-07-28-infinispan-server-tutorial.adoc:20]:
```suggestion
. Run Infinispan Server.
```

**Comment by karesti** [comment]
@oraNod done all

**Pull request merged**